backend/build.gradle.kts

```diff
@@ -128,3 +130,11 @@ tasks {
         }
     }
 }
+val compileKotlin: KotlinCompile by tasks
```

Comment:
proc je tu tohle?

Reply:
o par radek vyse je presne to same 
![image](https://user-images.githubusercontent.com/14038818/110951520-72fb8380-8345-11eb-9125-8f2b8365676e.png)

Reply:
asi zapomenuty copy paste